ui/src/features/annotator/tools/drawing-box-tool/crosshair/utils.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { PointerEvent, RefObject, useState } from 'react';
+
+import { Point } from '../../../types';
+
+interface UseCrosshair {
+    location: Point | null;
+    onPointerMove: (event: PointerEvent<SVGSVGElement>) => void;
+    onPointerLeave: (event: PointerEvent<SVGSVGElement>) => void;
+}
+
+type ElementType = SVGElement | HTMLDivElement;
+export const getRelativePoint = (element: ElementType, point: Point, zoom: number): Point => {
```

Comment:
Can we move this to a `tools/utils` so that all tools can use them? We will also need this in the other tools (see https://github.com/open-edge-platform/training_extensions/blob/develop/ui/src/features/annotator/hooks/use-translate.hook.ts#L26)

Reply:
Updated